src/test/java/com/spotify/autoscaler/AutoscaleJobTest.java

```diff
@@ -169,7 +154,12 @@ public void testUpperBound() throws IOException {
     final SemanticMetricRegistry registry = new SemanticMetricRegistry();
     job =
         new AutoscaleJob(
-            bigtableSession, stackdriverClient, cluster, db, registry, clusterStats, Instant::now);
+            bigtableSession,
+            stackdriverClient,
+            cluster,
+            db,
+            new AutoscalerMetrics(registry),
```

Comment:
Question, why sometimes we use the mocked `AutoscalerMetrics` and sometimes we create a new one?